src/nrtk_explorer/app/ui/image_list.py

```diff
@@ -4,36 +4,40 @@
 from trame_annotations.widgets.annotations import ImageDetection
 from nrtk_explorer.app.trame_utils import change_checker
 from nrtk_explorer.app.images.image_ids import get_image_state_keys
+from nrtk_explorer.widgets.nrtk_explorer import ScoreTable
 
 
 CSS_FILE = Path(__file__).with_name("image_list.css")
 
 COLUMNS = [
-    {"name": "id", "label": "Dataset ID", "field": "id", "sortable": True},
-    {"name": "truth", "label": "Original: Ground Truth Annotations", "field": "truth"},
-    {"name": "original", "label": "Original: Detection Annotations", "field": "original"},
+    {"name": "id", "label": "Dataset ID", "field": "id", "align": "left", "sortable": True},
     {
-        "name": "transformed",
-        "label": "Transformed: Detection Annotations",
-        "field": "transformed",
+        "name": "original",
+        "label": "Original",
+        "field": "original",
+        "align": "center",
+        "sortable": False,
     },
     {
-        "name": "original_ground_to_original_detection_score",
-        "label": "Ground Truth - Original Detection | Annotations Similarity",
-        "field": "original_ground_to_original_detection_score",
-        "sortable": True,
+        "name": "originalScores",
+        "label": "Annotations Similarity | Original",
+        "field": "originalScores",
+        "align": "center",
+        "sortable": False,
     },
     {
-        "name": "ground_truth_to_transformed_detection_score",
-        "label": "Ground Truth - Transformed Detection | Annotations Similarity",
-        "field": "ground_truth_to_transformed_detection_score",
-        "sortable": True,
+        "name": "transformed",
+        "label": "Transformed",
+        "field": "transformed",
+        "align": "center",
+        "sortable": False,
     },
     {
-        "name": "original_detection_to_transformed_detection_score",
-        "label": "Original Detection - Transformed Detection | Annotations Similarity",
-        "field": "original_detection_to_transformed_detection_score",
-        "sortable": True,
+        "name": "transformedScores",
+        "label": "Annotations Similarity | Transformed",
+        "field": "transformedScores",
+        "align": "center",
+        "sortable": False,
```

Comment:
Would be nice if we could sort by score still to find failure cases, but that extra credit. Could maybe just add more columns per model?